src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/entities/ccd/Event.java

```diff
@@ -140,6 +140,7 @@ public enum Event {
     UPDATE_TRIBUNAL_DECISION("updateTribunalDecision"),
     RECORD_REMISSION_REMINDER("recordRemissionReminder"),
     MARK_APPEAL_AS_REMITTED("markAppealAsRemitted"),
+    REFUND_CONFIRMATION("refundConfirmation"),
```

Comment:
Maybe we need to populate app.yml with this event for admin role?

Reply:
@ikirsanov it seemed to let me trigger it without it but I will add just in case